### ISSUE TITLE
Does this work with CouchDB? 

### ISSUE THREAD

**Issue by flimzy** [couchbase/go-couchbase#63]
Googling for CouchDB libraries for Go results in links to couch-go, which in turn all direct here. 

I'm guessing that this project has long since abandoned CouchDb support, am I correct? 

A small blurb in the Readme explaining why we're redirected here from couch-go, and whether or not this project is still relevant to CouchDB might be nice.

**Comment by tleyden** [comment]
Nope, this doesn't work with CouchDB, but you can check out https://github.com/dustin/go-couch -- I don't know if it's the best one, but I've used it extensively and it's worked well.
